### DISCUSSION TITLE
Environment Variables and Vcpkg updates

### DISCUSSION THREAD

**Discussion by slowik21861** [microsoft/vcpkg#19135] [Ideas]
I am using CMAKE as build system on macOS. Every project uses CMAKE_TOOLCHAIN_FILE provided by Vcpkg. After upgrading Vcpkg to newer version I have to modify .vscode/settings.json in every project to match new path to Vcpkg.cmake file

**Proposed solution**
One way to overcome this problem is to provide link "current version" or similar in constant path. Every upgrade of Vcpkg should update this link

**Comment by JackBoosY** [comment]
@dan-shaw Maybe we should do that, to ensure smoother version updates.
